data/updater.py

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env python2
+
```

Comment:
This is the update meant only for updating lepidopter, correct? If so, perhaps, I'd specify that with a toplevel comment (or docstring) in the file.